omod/src/test/java/org/openmrs/module/owa/web/controller/OwaRestControllerTest.java

```diff
@@ -138,4 +139,16 @@ public void install_wrongDownloadUrl() throws Exception {
 		List<App> appList = controller.install(requestData, request, response);
 		Assert.assertEquals("owa.not_a_zip", request.getSession().getAttribute(WebConstants.OPENMRS_ERROR_ATTR));
 	}
+
+	@Test
+	public void allowWebAdmin_shouldNotReturnNull() {
+		HttpServletRequest request = new MockHttpServletRequest(new MockServletContext(), "GET",
+		        "/rest/owa/allowModuleWebUpload");
+		HttpServletResponse response = new MockHttpServletResponse();
+		boolean allowModuleWebUpload = ModuleUtil.allowAdmin();
+		OwaRestController controller = (OwaRestController) applicationContext.getBean("owaRestController");
+		boolean allowAdmin = controller.allowWebAdmin(request, response);
```

Comment:
What is the value in calling controller.allowWebAdmin twice?

Reply:
the second call I am assigning the mock requests and response to it.

Reply:
this gets me the value which is asserting if its equal to the value of the module util

Reply:
Do you have to make  a second functional call?

Reply:
i have seen the repetition i was doing and i have fixed it   @dkayiwa